lib/icloud/session.rb

```diff
@@ -25,6 +24,15 @@ def initialize apple_id, pass, client_id=nil
       @client_id = client_id || default_client_id
     end
 
+    def pool
+      unless @pool
+        ensure_logged_in
```

Comment:
`get_startup` calls `ensure_logged_in`, so I don't think this line is necessary?